src/store/removeTool.js

```diff
@@ -0,0 +1,32 @@
+import { state } from './index.js';
+
+/**
+ * Removes all tools from the target element with the provided name
```

Comment:
This brings up the point, do we want to allow tools to have the same name? Seems a little dangerous with the way the current framework is set up. Although, I also can't see a reason why anyone would choose to name two tools the same on purpose.

Reply:
🤷‍♂️

_addTool_
```
  const toolAlreadyAddedToElement = getToolForElement(element, tool.toolName);

  if (toolAlreadyAddedToElement) {
    console.warn(
      `${tool.toolName} has already been added to the target element`
    );

    return;
  }
```

For the time being, we don't allow tool's with the same name to be added to the same element

Reply:
Oh, didn't realize that's already a thing! :)